amun/base/kustomization.yaml

```diff
@@ -9,6 +9,9 @@ resources:
   - argo-workflows/inspection-build-with-cpu.yaml
   - argo-workflows/inspection-run.yaml
   - argo-workflows/inspection-run-with-cpu.yaml
+  - argo-workflows/parse-inspection-inputs.yaml
+  - ../../../core/base/argo-workflows/send-message.yaml
```

Comment:
I think we want amun to be independent component, therefore send-message.yaml template need to be place under amun repo and not reference core. cc @harshad16

Reply:
yes, as @pacospace stated, we want amun to be independent component , so please move all amun related manifest under the amun directory.

Reply:
@harshad16 @pacospace Is it okay to use a symlink here?

Reply:
I know it is not convenient to maintain two copies, but I feel we should do it in this case.
Using symlink won't solve the independence we require for Amun.